docs-markdown/src/controllers/italic-controller.ts

```diff
@@ -1,106 +1,91 @@
-'use strict';
+"use strict";
 
-import { Range, Selection, TextEditorEdit, window } from 'vscode';
-import {
-	insertContentToEditor,
-	isMarkdownFileCheck,
-	isValidEditor,
-	noActiveEditorMessage,
-	postWarning,
-	showStatusMessage
-} from '../helper/common';
-import { insertUnselectedText } from '../helper/format-logic-manager';
-import { isBoldAndItalic, isItalic } from '../helper/format-styles';
-import { sendTelemetryData } from '../helper/telemetry';
+import { Range, Selection, TextEditorEdit, window } from "vscode";
+import { insertContentToEditor, isMarkdownFileCheck, isValidEditor, noActiveEditorMessage, postWarning, showStatusMessage } from "../helper/common";
+import { insertUnselectedText } from "../helper/format-logic-manager";
+import { isBoldAndItalic, isItalic } from "../helper/format-styles";
+import { sendTelemetryData } from "../helper/telemetry";
 
-const telemetryCommand: string = 'formatItalic';
+const telemetryCommand: string = "formatItalic";
 
 export function italicFormattingCommand() {
-	const commands = [{ command: formatItalic.name, callback: formatItalic }];
-	return commands;
+    const commands = [
+        { command: formatItalic.name, callback: formatItalic },
+    ];
+    return commands;
 }
 
 /**
  * Replaces current selection with MD italic formated selection
  */
 export function formatItalic() {
-	const editor = window.activeTextEditor;
-	if (!editor) {
-		noActiveEditorMessage();
-		return;
-	} else {
-		if (!isValidEditor(editor, true, 'format italic')) {
-			return;
-		}
+    const editor = window.activeTextEditor;
+    if (!editor) {
+        noActiveEditorMessage();
+        return;
+    } else {
+        if (!isValidEditor(editor, true, "format italic")) {
+            return;
+        }
 
-		if (!isMarkdownFileCheck(editor, false)) {
-			return;
-		}
+        if (!isMarkdownFileCheck(editor, false)) {
+            return;
+        }
 
-		// const selection = editor.selection;
-		// const selectedText = editor.document.getText(selection);
-		const selections: Selection[] = editor.selections;
-		let range;
+        // const selection = editor.selection;
+        // const selectedText = editor.document.getText(selection);
+        const selections: Selection[] = editor.selections;
+        let range;
 
-		// if unselect text, add italic syntax without any text
-		if (selections.length === 0) {
-			const cursorPosition = editor.selection.active;
-			const selectedText = '';
+        // if unselect text, add italic syntax without any text
+        if (selections.length === 0) {
+            const cursorPosition = editor.selection.active;
+            const selectedText = "";
```

Comment:
@bryce-turner I think these changes look good and we are just about ready to merge, but can you [install the eslint vscode extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) and save the files you have changed? It should automatically format and fix the linting errors. We recently switched to single quotes last week and I see double quotes in these files. You can also run this command to fix all the quotes `eslint . --ext .ts --fix` as well.